observability/logging/cluster-logging-deploying.adoc

```diff
@@ -47,10 +47,8 @@ include::modules/logging-loki-gui-install.adoc[leveloffset=+1]
 .Additional resources
 
 ifdef::openshift-enterprise,openshift-origin[]
-* xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN default CNI network provider]
 * xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes default Container Network Interface (CNI) network provider]
 * xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#ovn-k-network-policy[About OVN-Kubernetes network policy]
-* xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN default CNI network provider]
 * xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[About the OVN-Kubernetes default Container Network Interface (CNI) network provider]
```

Comment:
I'm not sure what happened here but it seems that there used to be 2 identical links to openshift-sdn and 2 identical links to ovn-kubernetes (bad link rot repair?), and you removed the 2 sdn links, but you could remove 1 of the ovn links as well?

Reply:
Good catch